Makefile

```diff
@@ -353,14 +353,14 @@ playwright:
 playwright-custom-components:
 	cd e2e_playwright; \
 	rm -rf ./test-results; \
-	pip_args="extra-streamlit-components streamlit-ace streamlit-antd-components streamlit-aggrid streamlit-autorefresh streamlit-chat streamlit-echarts streamlit-folium streamlit-lottie streamlit-option-menu streamlit-url-fragment";\
+	pip_args="extra-streamlit-components streamlit-ace streamlit-antd-components streamlit-aggrid streamlit-autorefresh streamlit-chat streamlit-echarts streamlit-folium streamlit-lottie streamlit-navigation-bar==3.1.2 streamlit-option-menu streamlit-url-fragment"; \
 	if command -v "uv" > /dev/null; then \
-		echo "Running command: uv pip install $(pip_args)"; \
-		uv pip install $(pip_args); \
+		echo "Running command: uv pip install $${pip_args}"; \
+		uv pip install $${pip_args}; \
 	else \
-		echo "Running command: pip install $(pip_args)"; \
-		pip install $(pip_args); \
-	fi;\
+		echo "Running command: pip install $${pip_args}"; \
+		pip install $${pip_args}; \
+	fi; \
```

Comment:
The installation seemed to have been broken since https://github.com/streamlit/streamlit/pull/8437 as the bash command variable substitution was not correct. Fixing here.

Reply:
Ups :( but why did the e2e test was marked as successful?

Reply:
It looks like a `wait_for_app_run` was missing in the popular_components test after the respective custom component was selected, which made the exception check succeed most of the time 😕 I have pushed an update to it